Dockerfile_llamacpp

```diff
@@ -48,15 +47,17 @@ FROM deps AS builder
 COPY --from=planner /app/recipe.json recipe.json
 RUN cargo chef cook \
     --recipe-path recipe.json \
-    --profile release-opt \
+    --profile release \
     --package text-generation-router-llamacpp
 COPY . .
 RUN cargo build \
-    --profile release-opt \
+    --profile release \
     --package text-generation-router-llamacpp --frozen
 
 FROM nvidia/cuda:12.8.0-cudnn-runtime-ubuntu24.04
+WORKDIR /app
 
+ENV DEBIAN_FRONTEND=noninteractive
 RUN apt update && apt install -y \
```

Comment:
`RUN apt update && apt upgrade -y && apt install -y \` otherwise CVE are flagged as critical in Amazon Inspector (for vulnerabilities scans)

Reply:
Done